knowledge-base/binaryformatter-exception-after-upgrade-to-net5.md

```diff
@@ -0,0 +1,48 @@
+---
+title: BinaryFormatter serialization and deserialization are disabled within this application is thrown in .NET5
+description: BinaryFormatter serialization and deserialization are disabled within this application after upgrade to .NET5
+type: troubleshooting
+page_title: BinaryFormatter serialization and deserialization are disabled within this application is thrown in .NET5
+slug: binaryformatter-exception-after-upgrade-to-net5
+position: 
+tags: .NET 5.0
+ticketid: 1494918
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+The following error: **Error registering the viewer with the service.
+An error has occurred.
+BinaryFormatter serialization and deserialization are disabled within this application** might be thrown after upgrade to .NET 5 in .NET Core or Blazor Projects.
+
+## Error Message
+Error registering the viewer with the service.
+An error has occurred.
+BinaryFormatter serialization and deserialization are disabled within this application. See https://aka.ms/binaryformatter for more information.
+
+## Cause\Possible Cause(s)
+We are currently working on building and distributing our packages built against  .NET 5.
+They are expected to be available in the R1 2021 Release of Reporting that is coming in the middle of January.
```

Comment:
I would say "They are expected to be available in the R1 2021 which is coming in the middle of January."